opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/_load.py

```diff
@@ -31,9 +30,32 @@
 _logger = getLogger(__name__)
 
 
+class _EntryPointDistFinder:
+    def __int__(self):
```

Comment:
Typo? `__int__` vs `__init__`

(Can probably just be deleted if https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2181/files#r1739126880 is accepted)

Reply:
Yes it is a typo